src/mbgl/util/mapbox.cpp

```diff
@@ -19,13 +19,13 @@ bool isMapboxURL(const std::string& url) {
 
 std::vector<std::string> getMapboxURLPathname(const std::string& url) {
     std::vector<std::string> pathname;
-    std::size_t startIndex = protocol.length();
-    std::size_t end = url.find_first_of("?#");
+    auto startIndex = protocol.length();
```

Comment:
FWIW, I prefer to use explicit types for normal local variables, rather than seeing `auto` and then needing to mentally work out what the type must be.